src/quick-setup.md

```diff
@@ -46,7 +46,7 @@ Bear in mind, if you plan on using the server purely for [standalone usage via O
 
 	![Calibration page](assets/img/quick_calibrate.png)
 
-	Please note: You should calibrate your trackers by following this same process every time you switch them on to use them!
+	Please note: Each time you turn your trackers on, they will automatically perform this calibration. make sure to leave them on a flat surface whille turning them on for each use of the trackers!
```

Comment:
```suggestion
	Please note: Each time you turn your trackers on, they will need to perform this calibration. Make sure to leave them on a flat surface after turning them on each time you want to use the trackers!
```
Pretty sure it can be any time after boot, but can double check with others